docs/cli-usage.md

```diff
@@ -126,34 +126,49 @@ djinni \
 
 ### Identifier Style
 
-Identifier styles (ex: `FooBar`, `fooBar`, `foo_bar`, `FOO_BAR`, `m_fooBar`)
-
-| Argument |
-| -------- |
-| `--ident-java-enum ...` |
-| `--ident-java-field ...` |
-| `--ident-java-type ...` |
-| `--ident-cpp-enum ...` |
-| `--ident-cpp-field ...` |
-| `--ident-cpp-method ...` |
-| `--ident-cpp-type ...` |
-| `--ident-cpp-enum-type ...` |
-| `--ident-cpp-type-param ...` |
-| `--ident-cpp-local ...` |
-| `--ident-cpp-file ...` |
-| `--ident-jni-class ...` |
-| `--ident-jni-file ...` |
-| `--ident-objc-enum ...` |
-| `--ident-objc-field ...` |
-| `--ident-objc-method ...` |
-| `--ident-objc-type ...` |
-| `--ident-objc-type-param ...` |
-| `--ident-objc-local ...` |
-| `--ident-objc-file ...` |
-
```

Comment:
I've replaced this listing with the one that includes the default styles and is split up into languages. I think there is no need to document all values twice.